cc-check.go

```diff
@@ -69,7 +69,11 @@ var requiredKernelModules = map[string]kernelModule{
 	"kvm_intel": {
 		desc: "Intel KVM",
 		parameters: map[string]string{
-			"nested":             "Y",
+			"nested": "Y",
+			// "VMX Unrestricted mode support". This is used
```

Comment:
@grahamwhaley unrestricted mode is used by KVM to directly handle execution of the VM when it is in real and protected mode. We need this to eliminate the use of emulation. There is no co-relation between this and virtio 1.0. We do not have support for emulation in q35-lite and pc-lite. So checking for this is still mandatory.